.github/CODEOWNERS

```
@@ -327,7 +327,8 @@
 /core/templates/services/internet-connectivity.service*.ts @oppia/frontend-infrastructure-reviewers
 /core/templates/services/server-connection-backend-api.service*.ts @oppia/frontend-infrastructure-reviewers
 /core/templates/services/number-conversion.service*.ts @oppia/frontend-infrastructure-reviewers
-
+/core/templates/services/insert-script.service.ts @oppia/frontend-infrastructure-reviewers
+/core/templates/services/insert-script.service.ts @oppia/core-reviewers
```

I think @srijanreddy98 asked for both frontend-infrastructure-reviewers and core-reviewers to be code-owners for this file (i.e. either one can approve changes). However, this code will make core-reviewers the only code owner because later lines override earlier lines in CODEOWENRS files. To make both groups code-owners, you need this:

```suggestion
/core/templates/services/insert-script.service.ts @oppia/frontend-infrastructure-reviewers @oppia/core-reviewers
```